### DISCUSSION TITLE
آپشن تنظیم آسان ترتیب هاست ها

### DISCUSSION THREAD

**Discussion by SheykhDavid** [Gozargah/Marzban#592] [Ideas]
بنظرم خیلی خوب میشد قابلیتی اضافه بشه که با شماره گذاری یا درگ هاست ها میشد ترتیب اونها رو برای نمایش یوزر تنظیم کرد

**Comment by erfjab** [comment]
> بنظرم خیلی خوب میشد قابلیتی اضافه بشه که با شماره گذاری یا درگ هاست ها میشد ترتیب اونها رو برای نمایش یوزر تنظیم کرد

این مورد رو باید ایشو کنید عزیز که در حال بررسی و اضافه شدن می‌باشد.
